tools/ports/icu.py

```diff
@@ -78,14 +78,16 @@ def create_libicu_i18n(lib_output):
   return [
       shared.cache.get_lib(get_lib_name(libname_libicu_common, settings), create_libicu_common), # this also prepares the build
       shared.cache.get_lib(get_lib_name(libname_libicu_stubdata, settings), create_libicu_stubdata),
-      shared.cache.get_lib(get_lib_name(libname_libicu_i18n, settings), create_libicu_i18n)
+      shared.cache.get_lib(get_lib_name(libname_libicu_i18n, settings), create_libicu_i18n),
+      shared.cache.get_lib(get_lib_name(libname_libicu_io, settings), create_libicu_io)
```

Comment:
It doesn't look like `create_libicu_io` is defined in this PR?

Reply:
sorry
fixed